libqtile/core/manager.py

```diff
@@ -1474,7 +1474,7 @@ def f(cmd):
                     logger.info('No command entered.')
                     return
                 try:
-                    result = eval(u'c.{0:s}'.format(cmd))
+                    result = eval(u'c.{0:s}'.format(cmd))  # pylint: disable=eval-used
```

Comment:
This one it feels like we can just delete (i.e. delete `cmd_qtilecmd`). Nobody is (successfully) typing the qtile API directly into their prompt widget on their bar...